joint_state_controller/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(joint_state_controller)
 
 # Default to C++14
```

Comment:
Best to remove the useless (and now incorrect) comment.

Reply:
:+1: